docs-devsite/vertexai-preview.vertexaierror.md

```diff
@@ -0,0 +1,76 @@
+Project: /docs/reference/js/_project.yaml
+Book: /docs/reference/_book.yaml
+page_type: reference
+
+{% comment %}
+DO NOT EDIT THIS FILE!
+This is generated by the JS SDK team, and any local changes will be
+overwritten. Changes should be made in the source code at
+https://github.com/firebase/firebase-js-sdk
+{% endcomment %}
+
+# VertexAIError class
+Error class for the Vertex AI for Firebase SDK.
+
+<b>Signature:</b>
+
+```typescript
+export declare class VertexAIError extends FirebaseError 
+```
+<b>Extends:</b> [FirebaseError](./util.firebaseerror.md#firebaseerror_class)
+
+## Constructors
+
+|  Constructor | Modifiers | Description |
+|  --- | --- | --- |
+|  [(constructor)(code, message, customErrorData)](./vertexai-preview.vertexaierror.md#vertexaierrorconstructor) |  | Creates a new VertexAIError instance. |
+
+## Properties
+
+|  Property | Modifiers | Type | Description |
+|  --- | --- | --- | --- |
+|  [code](./vertexai-preview.vertexaierror.md#vertexaierrorcode) |  | [VertexAIErrorCode](./vertexai-preview.md#vertexaierrorcode) |  |
+|  [customErrorData](./vertexai-preview.vertexaierror.md#vertexaierrorcustomerrordata) |  | [CustomErrorData](./vertexai-preview.customerrordata.md#customerrordata_interface) \| undefined |  |
+|  [message](./vertexai-preview.vertexaierror.md#vertexaierrormessage) |  | string |  |
+
+## VertexAIError.(constructor)
+
+Creates a new VertexAIError instance.
```

Comment:
same here... can we wrap VertexAIError with backticks or something?

Reply:
Yeah absolutely- fixed [here](https://github.com/firebase/firebase-js-sdk/pull/8240/commits/a6b28b5bf21f63027ba8408f8dc3eeb5c35a415c?diff=unified&w=0#diff-b411e0790dceb6354c3392c8523db0cecff6cc6b010e9fc416018c640fa8286aR38)